Updates/5430_Ulduar_Hodir.sql

```diff
@@ -0,0 +1,47 @@
+-- Ulduar: Boss encounter Hodir
+DELETE FROM `creature_spell_list` WHERE `Id`  IN (3284500, 3284600);
+INSERT INTO `creature_spell_list` (`Id`, `Position`, `SpellId`, `InitialMin`, `InitialMax`, `RepeatMin`, `RepeatMax`, `Flags`, `TargetId`, `ScriptId`, `Availability`, `Probability`, `Comments`) VALUES
+(3284500, 0, 61968, 50000, 50000, 40000, 40000, 2,   0,     0, 100, 1, 'Hodir 10 - Flash Freeze'),
+(3284500, 1, 62478, 70000, 70000, 60000, 60000, 0,   0, 29281, 100, 1, 'Hodir 10 - Frozen Blows'),
+(3284500, 2, 62469, 25000, 30000, 15000, 15000, 2, 100,     0, 100, 1, 'Hodir 10 - Freeze'),
+(3284600, 0, 61968, 50000, 50000, 40000, 40000, 2,   0,     0, 100, 1, 'Hodir 25 - Flash Freeze'),
+(3284600, 1, 63512, 70000, 70000, 60000, 60000, 0,   0, 29281, 100, 1, 'Hodir 25 - Frozen Blows'),
+(3284600, 2, 62469, 25000, 30000, 15000, 15000, 2, 100,     0, 100, 1, 'Hodir 25 - Freeze');
+
+DELETE FROM `creature_spell_list_entry` WHERE `Id` IN (3284500, 3284600);
+INSERT INTO `creature_spell_list_entry` VALUES
+(3284500, 'Ulduar - Hodir (10)', 100, 100),
+(3284600, 'Ulduar - Hodir (25)', 100, 100);
+
+DELETE FROM `string_id` WHERE `Name`='ULDUAR_HODIR_HELPERS';
+INSERT INTO `string_id` VALUES
+(6030701,'ULDUAR_HODIR_HELPERS');
+
+DELETE FROM `string_id` WHERE `Name`='ULDUAR_HODIR_FIRES';
+INSERT INTO `string_id` VALUES
+(6030702,'ULDUAR_HODIR_FIRES');
+
+UPDATE `creature_template` SET `SpellList`=3284500 WHERE `entry`=32845; -- Hodir (10)
+UPDATE `creature_template` SET `SpellList`=3284600 WHERE `entry`=32846; -- Hodir (25)
+-- Snowpacked Icicle Target
+UPDATE `creature_template` SET `StaticFlags1`=`StaticFlags1`|0x00000100|0x02000000 WHERE `entry`=33174;
+UPDATE `creature_template` SET `ScriptName`='' WHERE `entry`=33174;
```

Comment:
this goes into core updates. It will be overwrittren by core updates if added here.